### ISSUE TITLE
JWT Verifier not accessing JWKS file?

### ISSUE THREAD

**Issue by Yudy186** [davidgtonge/jwt#47]
Not sure if this project is active at all... was steered to it by someone well-versed in JWT's (in the context of OAuth). It's been quite useful in confirming various things as I've been learning OAuth. Thank you

I could have sworn the site was working fine 6 months or so ago; I just came back to it and am having an issue with using my jwks (url below)
- A Key not found error is displayed on the right side of the page
- There are no keys displayed on the left hand beneath the JWKS input field (your sample endpoints like accounts.google.com do displays the keys found), 
- in the Network tab of the the Developers Tools window, it appears the proxy is returning a 404. 
However, I can access the jwks page it just fine from the address bar.  This issue originated when my own app encountered something similar when I requested an access token ... I get a 400 from the server -- invalid request with a description of "JWS signature invalid", which could also be a result of the jwks not being accessed properly. I assumed it was my error, until I plugged the requesting token into your site and obtained the above results

The jwks in question is: 
   https://dev3.turingscraft.com/codelab/api/lti3/actions/jwks
however, I also looked up and found some other public jwks site, notably:
   https://api.apps.cam.ac.uk/oauth2/v1/.well-known/jwks.json
and the same issue occurs ... i.e., no keys seem to be found (or at least that's your diagnostic -- though the same 404 occurs seemingly implying you're not getting the file whatsoever).

In case you ARE listening to this project still, and have any inclination to pursue this, I can also forward a sample token, though the 404 seems to preclude the issue being the contents of the token.

**Comment by davidgtonge** [comment]
The proxy deployment had a problem, I've fixed it, so please try again

**Comment by Yudy186** [comment]
Wow… that WAS fast! I wasn’t even sure I’d get any response. In any event, it’s a big help (I think) … the cam.ac.uk <http://cam.ac.uk/> wks site is now being properly retired and parsed by your page, while mine JWKS is still producing a 404 from your proxy … so I’m assuming there’s something environmental going on on my side (as I said, I can access my JWKS page from a browser, and I believe from one of my local standalone apps). 

Sorry for having bothered you; I was quite hesitant to create an Issue — being a newbie at JWT-related stuff, I was concerned I was ‘crying wolf’ over some dumb mistake I had made. It was only once I saw a second unrelated JWKS page failing to be parsed by you, that I took the plunge. And for all I know, it’s still a dumb error on my part.
 
Thanks again, and have a pleasant and productive New Year.

	— Gerald
